### ISSUE TITLE
Portable TaskExpolorer

### ISSUE THREAD

**Issue by alabotski** [DavidXanatos/TaskExplorer#36]
Hi I download zip file
I thought it is normal portable version

But TaskExplorer.ini file creates in c:\Users\XXX\AppData\Local\TaskExplorer\
Not in main directory

**Comment by DavidXanatos** [comment]
ti use it filly portable you need to create a TaskExplorer.ini in the pap dirctroy that it will be used and non created in appdata

**Comment by alabotski** [comment]
Currently only exe file is available for download. Where can I get zip?

**Comment by DavidXanatos** [comment]
@Mitch037 you are right I'll port the sbie installer script that should be the best solution, it will be improved in the next build
